### DISCUSSION TITLE
How to identify if a radio button is selected or not ? or alternatively how to identify if a radioButton has been selected.

### DISCUSSION THREAD

**Discussion by harimadusumilli** [getgauge/taiko#1966] [General]
I am using Taiko along with Gauge against an Angular App.

I  can do radioButton('text').select() to click a specific radio button.

But in my case lets say the page is loaded with a pre-checked radio button and  I need to know if it has been checked or not ? Is there an API that can do it. Currently I don't think it can. I tried looking at various attributes to do something like radioButton({id:'myButton', class:'someClass'}).exists() it returns true if the attributes are present, but don't know how to identify if the radioButton is selected or not.

Any suggestion or advise will be helpful.

Thank you.

**Comment by NivedhaSenthil** [comment]
`await radioButton('text').isSelected()` should say if the radioButton is selected or not. Please refer [radioButtonWrapper](https://docs.taiko.dev/api/radiobuttonwrapper/) for more info.

**Reply by harimadusumilli** [reply]
Thank you for your response. I somehow seemed to have missed out on looking at the radioButton wrapper documentation. I am going ahead and closing this discussion topic of mine.
